gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/SettingsCategoryPanel.java

```diff
@@ -86,6 +86,21 @@ public SettingsCategoryPanel(final String id, final IModel<GbSettings> model, fi
 		this.expanded = expanded;
 	}
 
+	public enum DropKeepUsage {
```

Comment:
If only used in this class this could be private?

```suggestion
	private enum DropKeepUsage {
```

Reply:
Sure; done.